engine/src/main/java/org/hibernate/search/v6poc/search/dsl/predicate/SearchPredicateContainerContext.java

```diff
@@ -20,7 +20,7 @@
  */
 public interface SearchPredicateContainerContext<N> {
 
-	AllPredicateContext<N> all();
+	MatchAllPredicateContext<N> all();
```

Comment:
Maybe we should rename this to `matchAll()` too... But I guess it's a different story, since it's user API. I'll merge anyway.